Sources/TensorFlow/Initializers.swift

```diff
@@ -499,3 +499,54 @@ public extension Tensor where Scalar: BinaryFloatingPoint,
         self = Tensor.glorot(fromStandardUniform: uniform, shape: shape)
     }
 }
+
+fileprivate extension Tensor where Scalar: BinaryFloatingPoint {
+    private static func glorot(
+        fromStandardNormal normalUniform: __shared Tensor<Scalar>,
```

Comment:
I don't think this argument tag and name represent what this argument is. Maybe something like this is better:
```swift
fromStandardNormalScaledBy varianceScalingFactor: __shared Tensor<Scalar>,
```
Even though it's long I believe it better represents what the argument is.

Reply:
done